src/backend/distributed/commands/distribute_object_ops.c

```diff
@@ -88,7 +88,7 @@ static DistributeObjectOps Any_AlterFunction = {
 static DistributeObjectOps Any_AlterObjectSchema = {
 	.deparse = NULL,
 	.qualify = NULL,
-	.preprocess = PreprocessAlterTableSchemaStmt,
+	.preprocess = NULL,
```

Comment:
this struct is now the same as `NoDistributeOps`. Can we remove this struct and remove where we return this in favour of `NoDistributeOps`.